src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/SemanticChecker.cs

```diff
@@ -288,46 +288,7 @@ private ACCESSERROR CheckAccessCore(Symbol symCheck, AggregateType atsCheck, Sym
 
         public bool CheckBogus(Symbol sym)
```

Comment:
Please use this everywhere this functionality is needed. Example: GroupToArgsBinder.LookForCandidates()